### ISSUE TITLE
Create Single Canvas type Callout

### ISSUE THREAD

**Issue by larssondenise** [alkem-io/client-web#2855]
## Description
As an Admin, I want to be able to select a template for every new Canvas on a Callout. So that I have more structured Canvas callouts. Also, I want an easy way of creating a callout with a clear vision on what the different types are. 
_Example: I want users to share personas via the template that I have made._

As a user I want a simple way to create a new card

## Acceptance criteria
Callout create:
- [x] Clicking on Canvases will open the option to chose a Canvas template from the Canvas templates at Hub level
- [x] Update Permission section text

Canvas Create:
- [x] CanvasTemplate to be picked up directly from the Callout entity when adding a new contribution
- [ ] Visuals for the Canvas are picked up from the CanvasTemplate (?) 

Server:
- [x] Callout entity extended to allow selecting a CanvasTemplate
- [x] Update existing dto's
- [x] Migration to add the additional columns to the Callout entity

TO WORK OUT FURTHER Migration of data:
- [ ] Every callout of type Canvas gets a canvasTemplate instance created and linked
- [ ] for callouts whose nameID starts with "othercards":: Type = "contribution", defaultDescription = "Please share your contribution. The more details the better!", visual=same as current "other" visual (if other was renamed then default to empty visual)
- [ ] for callouts with no cards, create cardTemplate with: Type = '', defaultText = '', visual=same as current "other" visual
- [ ] for callouts with a single card type used across all cards: Type = the card type, defaultText = '', visual=same as current "other" visual
- [ ] for callouts with multiple card types used: Type = '', defaultText = '', visual=same as current "other" visual

## Additional Context
https://www.figma.com/file/olZkcLJKpUYRMjakhfGpB8/Callouts?node-id=568%3A1687

**Comment by techsmyth** [comment]
@alstojanovic for all existing canvas callouts a blank canvas should be added that is the canvas contained by the Canvas Template. So migration can be very simple: just create authorization, create canvas template with empty canvas (just grab json from a graphql call) etc. If the admin later wants to set a particular canvas then they can do that via settings. 

Only question I have is how the setting of the canvas value works with migrations now that the canvas is encoded? Can we encode a raw excallidraw json file in the migration script?

**Comment by alstojanovic** [comment]
@techsmyth yes, all doable. The empthy canvas encoding can be generated and written as the value.
